package.json

```diff
@@ -93,6 +93,7 @@
         "@sendbird/chat": "^4.9.7",
         "@types/react-transition-group": "^4.4.4",
         "babel-jest": "^27.3.1",
+        "builtin-modules": "^3.3.0",
```

Comment:
This is not required. Just need to update this branch with `feature/wallets_with_traders_hub_2`